src/components/OutsideClickHandler.jsx

```diff
@@ -18,21 +19,18 @@ export default class OutsideClickHandler extends React.Component {
   }
 
   componentDidMount() {
-    if (document.addEventListener) {
-      // `useCapture` flag is set to true so that a `stopPropagation` in the children will
-      // not prevent all outside click handlers from firing - maja
-      document.addEventListener('click', this.onOutsideClick, true);
-    } else {
-      document.attachEvent('onclick', this.onOutsideClick);
-    }
+    // `capture` flag is set to true so that a `stopPropagation` in the children
+    // will not prevent all outside click handlers from firing - maja
+    this.clickHandle = addEventListener(
```

Comment:
Here's where it would go: https://github.com/lencioni/consolidated-events/blob/603e9c9c54f3367cf4ca401672447e66d0731fcd/src/TargetEventHandlers.js#L44